modules/serverless-nodejs-functions-context-objects.adoc

```diff
@@ -53,3 +53,64 @@ function handle(context, data)
 ----
 
 The `data` parameter in this example is a JavaScript object that contains the `customerId` and `productId` properties.
+
+[id="serverless-nodejs-functions-context-objects-arbitrary-data_{context}"]
```

Comment:
This is probably an opportunity to modularize.

Reply:
I'd also lean towards splitting this, but this is currently an (unwritten) convention for this type of reference docs in Serverless. It's better to do a full sweep for all such modules separately.